### PR TITLE
Story2 see list of planets

### PR DESCRIPTION
**Pull request by cfbuckley14**

### CODE REVIEW

File: src/test/PlanetTest.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace PlanetApp\Test;
+use PHPUnit\Framework\TestCase;
+use PlanetApp\Planet;
+
+//require __DIR__ . '/../Planet.php';
```
Comment (cfbuckley14) [inline src/test/PlanetTest.php:7]: remove comment

File: style.scss [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+h1 {
+  display: none;
+}
+h2 {
+  font-size: 3rem;
+  margin: 0;
+}
+
```
Comment (cfbuckley14) [inline style.scss:15]: fix spacing

File: src/Planet.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        <div class="planet-card">
+            <img src="images/'.$this->image_src.'">
+            <div class="planet-card-info-container">
+                <h2>'.$this->name.'</h2>
+                <h3>'.$this->type.'</h3>
+            </div>
+        </div>
+        ');
+    }
+
```
Comment (RichVelo) [inline src/Planet.php:38]: space after method - other methods, in other php files don't have spaces so I think we should be doing the same over all files and methods. either would be fine imo

File: style.scss [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      filter: drop-shadow(0 0 50px #ef5f02);
+    }
+    50% {
+      filter: drop-shadow(0 0 50px #c31b02);
+    }
+    100% {
+      filter: drop-shadow(0 0 50px #ef5f02);
+    }
+  }
+
```
Comment (RichVelo) [inline style.scss:50]: I think we could remove these spaces, not sure they are needed

File: src/PlanetHydrator.php [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<?php
+namespace PlanetApp;
+
+class PlanetHydrator
+{
+
```
Comment (RichVelo) [inline src/PlanetHydrator.php:6]: space?

**Pull request closed without merging**